charities.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+
+// Create Charities Schema
+
+const CharitySchema = new mongoose.Schema({
+  charity: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  image: {
+    type: String, // string to URL?
```

Comment:
For now yes. Down the line we can get an upload form for the admin side and actually store the image binary on the db.